### DISCUSSION TITLE
When the search shows too many items, you cannot switch to the next page.

### DISCUSSION THREAD

**Discussion by aaro-n** [requarks/wiki#3605] [Error / Bug Report]
I am using macbook pro 13 2018, with a display resolution of 2560 x 1600.
When searching, if there are too many entries, you cannot switch.
Phenomenon reproduced: https://docs.requarks.io/, search for the keyword "wiki", display 49 items, scroll up and down the browser, no change.
<img width="1346" alt="123" src="https://user-images.githubusercontent.com/45537680/89721498-db316200-da10-11ea-9009-de7bc7944431.png">

**Comment by Jhobean** [comment]
This ? If yes, its a duplicate issue.

https://github.com/Requarks/wiki/pull/2232

**Comment by aaro-n** [comment]
Is the same problem

**Comment by Smankusors** [comment]
Duplicate of #1672
